lang_java/tree_sitter/grammar_to_tree_sitter_ast.py

```diff
@@ -0,0 +1,111 @@
+import json
+
+with open("java_grammar.json", 'r') as fs:
```

Comment:
When you convert the script to use node-types.json as discussed, this blacklisting will become way simpler!